ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -2944,11 +2946,12 @@
 			baseConfigurationReference = D047263419E49FE8006002AA /* iOS-Framework.xcconfig */;
 			buildSettings = {
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				"CODE_SIGN_IDENTITY[sdk=iphonesimulator*]" = "";
+				"CODE_SIGN_IDENTITY[sdk=iphonesimulator*]" = "iPhone Developer";
```

Comment:
Some unnecessary changes (like this one) were made to the project file. Please undo these changes.